refactor: es6 modules & testable workshop (#52)
KILTprotocol/docs: docs/sdk/1-workshop/04-claimer/01-did.md

Take a look at our [DID documentation](/docs/sdk/core-feature/did) if you want to learn more about DIDs and the difference between their light and full versions.

## Generate Keys

> Similar to the attester, the claimer too has to setup the DID keys.

Reviewer: ```suggestion
Similar to the attester, the claimer must set up the DID keys.
```
